editor/components/post-publish-panel/toggle.js

```diff
@@ -64,7 +49,12 @@ function PostPublishPanelToggle( {
 			disabled={ ! isButtonEnabled }
 			isBusy={ isSaving && isPublished }
 		>
-			{ isBeingScheduled ? __( 'Schedule…' ) : __( 'Publish…' ) }
+			{ ! hasPublishAction && (
+				isBeingScheduled ? __( 'Schedule…' ) : __( 'Submit for Review…' )
+			) }
+			{ hasPublishAction && (
+				isBeingScheduled ? __( 'Schedule…' ) : __( 'Publish…' )
+			) }
```

Comment:
Minor: Do you think if would be clearer if we avoid ternaries here:

```js
{ isBeingScheduled && __( 'Schedule…' ) }
{ ! isBeingScheduled &&  hasPublishAction && __( 'Publish…' ) }
{ ! isBeingScheduled && ! hasPublishAction && __( 'Submit for Review…' ) }
```

Reply:
Thank you so much for reviewing this @youknowriad! 

This looks way clearer to me. Test in progress and another change incoming...

Reply:
Done [here](https://github.com/WordPress/gutenberg/pull/6729/commits/8f7c3f74f3dffa3590a54ddf1335edcc47d86e32).

I had initially tried to nest ternaries but Travis didn't like it. But this looks way nicer. Thank you for the recommendation @youknowriad.